### PR TITLE
Angus/refactoring

### PR DESCRIPTION
**Pull request by veggiesaurus**
Refactoring: moves per-Frame stores to a `stores/Frame` subfolder. The rest of the changes are just adjusting the imports.

Making @kswang1029 the assigned reviewer, as we mainly just need some user testing to make sure nothing breaks with this branch :smile:

### CODE REVIEW

File: src/models/LayoutConfig.ts [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import Ajv from "ajv";
 import {AppStore, WidgetConfig, CatalogStore} from "stores";
 import {PresetLayout} from "models";
-import {findDeep, smoothStepOffset} from "utilities";
+import {smoothStepOffset} from "utilities/math";
```
Comment (veggiesaurus) [inline src/models/LayoutConfig.ts:4]: this seems to be necessary to prevent a circular dependency

**Review by kswang1029 — APPROVED** [review]
All current e2e tests are passing. I assume this is fine. 👍

File: src/models/CatalogControlMap.ts [hunk 2]
```diff
@@ -1,5 +1,7 @@
 import * as AST from "ast_wrapper";
-import {FrameStore, CatalogStore} from "stores";
+import {CatalogStore} from "stores";
+import {FrameStore} from "stores/Frame";
+
```
Comment (YuHsuan-Hwang) [inline src/models/CatalogControlMap.ts]: remove extra line

File: src/stores/Frame/index.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+export * from "./ColorbarStore";
+export * from "./ContourStore";
+export * from "./ContourConfigStore";
+export * from "./ContourConfigStore";
```
Comment (YuHsuan-Hwang) [inline src/stores/Frame/index.ts]: remove duplicate line

**Review by YuHsuan-Hwang — CHANGES_REQUESTED** [review]

**Review by YuHsuan-Hwang — APPROVED** [review]

**Pull request merged**